runelite-client/src/main/java/net/runelite/client/plugins/dailytaskindicators/DailyTasksPlugin.java

```diff
@@ -99,29 +98,48 @@ else if (event.getKey().equals("showEssence"))
 			{
 				hasSentEssenceMsg = false;
 			}
+			else if (event.getKey().equals("showSand"))
+			{
+				hasSentSandMsg = false;
+			}
+			else if (event.getKey().equals("showEcto"))
+			{
+				hasSentEctoMsg = false;
+			}
 		}
 	}
 
 	@Subscribe
-	public void onGameStateChanged(GameStateChanged event)
+	public void onGameTick(GameTick event)
```

Comment:
Since these are all varbit related, why not use the varbit change event?

Reply:
Oh I forgot to mention this, sorry. I don't use the varb change event because then all the check methods would return true because all the values are 0.

Reply:
Store the old value and then check the new value to see if it has changed.